src/main/java/seedu/duke/command/task/AddTaskCommand.java

```diff
@@ -47,6 +54,31 @@ public void execute() {
         Ui.showToUserLn("Task successfully created.");
         Ui.showToUserLn(addedTask.toString());
     }
+
+    public boolean doesTitleExist(String title) {
+        boolean titleAlreadyExist = false;
+        if (projectManager.isEmpty()) {
+            return false;
+        }
+        for (int i = 1; i <= projectManager.getSelectedProject().getProjectBacklog().size(); i++) {
```

Comment:
Maybe try iterating through something like `for (Task task: projectmanager.getSelectedProject().getProjectBacklog()` ?

Reply:
Will look into this and implement if possible.

Reply:
Implemented.